local.properties

```diff
@@ -0,0 +1 @@
+sdk.dir=ANDROID_SDK_ROOT
```

Comment:
shouldn't be committed to git, should be generated dynamically on CI if required

Reply:
I didn't find any example to generate it dynamically, do you know how it can be done?

Reply:
Here's the error I get on `app:lint` :
```
Use '--warning-mode all' to show the individual deprecation warnings.
* What went wrong:
A problem occurred configuring project ':playground'.
> /home/runner/work/soni-sori/soni-sori/local.properties (No such file or directory)
```